feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -170,7 +192,7 @@ class PreviewViewModel @Inject constructor(
                     )
                 )
                 // apply to cameraUseCase
-                cameraUseCase.flipCamera(previewUiState.value.currentCameraSettings.isFrontCameraFacing)
+                cameraUseCase.setLensFacing(cameraUseCase.config.value.lensFacing.next())
```

Comment:
If using the `next()` extension function, do we still need the `isFacingFront` boolean passed in to this method? Also, if we end up adding another lens facing enum, such as EXTERNAL, then `flipCamera` might not be the best name for this method since we could be switching between 3 cameras.